pyproject.toml

```diff
@@ -18,13 +18,13 @@ dependencies = [
     "scikit-image", # secretly required by intake-xarray as default reader
     "torch",
     "xarray",
-    "resnet50-cefas@git+https://github.com/jmarshrossney/resnet50-cefas",
+    #"resnet50-cefas@git+https://github.com/jmarshrossney/resnet50-cefas",
```

Comment:
`conda-lock` fails here. According to the [documentation](https://conda.github.io/conda-lock/src_pyproject/#pure-pip-dependencies) it can deal with this situation where the `poetry` dependency specification is adopted, but doesn't mention how to resolve this for [PEP 621](https://peps.python.org/pep-0621/#dependencies-optional-dependencies) style `pyproject.toml` files.

This might be to do with `conda-lock` using `poetry` under the hood..